sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/GcpApiSurfaceTest.java

```diff
@@ -91,6 +91,20 @@ public void testGcpApiSurface() throws Exception {
             Matchers.<Class<?>>equalTo(com.google.cloud.ByteArray.class),
             Matchers.<Class<?>>equalTo(com.google.cloud.Date.class),
             Matchers.<Class<?>>equalTo(com.google.cloud.Timestamp.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Identity.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Identity.Type.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Policy.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Policy.Builder.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Policy.Marshaller.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Policy.DefaultMarshaller.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.Role.class),
+            Matchers.<Class<?>>equalTo(com.google.cloud.StringEnumValue.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.Binding.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.Binding.Builder.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.Policy.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.Policy.Builder.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.BindingOrBuilder.class),
+            Matchers.<Class<?>>equalTo(com.google.iam.v1.PolicyOrBuilder.class),
```

Comment:
I had to add these classes in whitelist. Details in [BEAM-8758](https://issues.apache.org/jira/browse/BEAM-8758?focusedCommentId=17029981&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17029981).

I hope they are acceptable.